src/pages/ProfilePage/views/FavoritesView/FavoritesView.tsx

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import Grid from '@mui/material/Grid';
+import { readAll } from 'src/firebase';
+import { useState, useEffect } from 'react';
+import FavoriteHouses from '../../../../components/ProfilePage/FavoriteHouses/FavoriteHouses';
+
+const FavoritesView = () => {
+  const [favorites, setFavorites] = useState<BasicHouseData[]>([]);
+
+  // GET DATA FROM FIREBASE (TRY OUT) - DELETE AFTER SUCCESSFULLY PUTTING DATA ON THE SERVER
+  const fetchHouses = async () => {
+    const fetchedHouses = await readAll('houses');
+    setFavorites(() => fetchedHouses);
+  };
+
+  useEffect(() => {
+    fetchHouses();
+  }, []);
+
+  // AXIOS FETCH DATA FROM SERVER - USE IT WHEN IT ALL WORKS
+  // const url = 'http://localhost:4000/users/my-favorites';
+
+  // useEffect(() => {
+  //   getFavorites();
+  // }, [])
+
+  // const getFavorites = () => {
+  //   axios.get<BasicHouseData[]>(`${url}`)
+  //     .then((res) => {
+  //       const allFavorites = res.data;
+  //       setFavorites(allFavorites);
+  //     })
+  //     .catch((err) => console.error(`Error: ${err}`));
+  // };
```

Comment:
Are you planning to do something with the commented part as part of this Pull Request?

Reply:
Well, I thought we want to fetch this data from a server, I placed the firebase houses only as dummy data to see if styling is allright

Reply:
Sure. You have a separate task for it anyway. If you use the changes that are now on `develop`, you could do it consistently with the rest of the new requests using the hook provided by Marta.